File: tests/fixtures/languages/sample.cpp
// Minimal C++ fixture for language parse-validation tests.

namespace demo {

class Greeter {
public:
    explicit Greeter(int n) : n_(n) {}
    int value() const { return n_; }

private:
    int n_;
};

int free_function(int x) {
    return x + 1;
}

}  // namespace demo

File: tests/fixtures/languages/Sample.cs
// Minimal C# fixture for language parse-validation tests.

using System;

namespace Demo;

public class Greeter
{
    private readonly string _label;

    public Greeter(string label)
    {
        _label = label;
    }

    public string Greet() => $"Hello, {_label}";
}

public static class Util
{
    public static int Add(int a, int b) => a + b;
}

public interface IDoubler
{
    int Doubled();
}
